packages/wonder-stuff-render-server/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "exclude": [
+    "dist"
+  ],
+  "extends": "../tsconfig-shared.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "src",
+  },
+  "references": [
+    {"path": "../wonder-stuff-core"},
+    {"path": "../wonder-stuff-server"}
+  ]
+}
```

Comment:
**suggestion:** `tsconfig-build.json` at the root needs to be updated with all new packages so that the .d.ts files will get created when running `yarn build:types`.

Reply:
This is done in the next PR.